### ISSUE TITLE
predict() has a missing step label in inline comments (step 2 skipped)

### ISSUE THREAD

**Issue by luipin** [luipin/set-ml-module#24]
### Summary
The `predict()` function in `src/models/predictor.py` has numbered inline section comments, but the numbering jumps from step 1 to step 3, skipping step 2 entirely.

### Code location
`src/models/predictor.py`, lines 63–69:

```python
# --- 1. Load and preprocess ---
transform = get_val_transforms()
tensor = transform(image=_load_image(image))["image"]
batch = tensor.unsqueeze(0)

# --- 3. Inference ---    <-- should be "2. Inference"
device = next(model.parameters()).device
```

### Fix
Change `# --- 3. Inference ---` to `# --- 2. Inference ---` and renumber the subsequent section (`# --- 4. Convert logits ---`) to `# --- 3. Convert logits ---`.

### Files affected
- `src/models/predictor.py`
